### PR TITLE
Updated Polish translation

### PR DESCRIPTION
**Pull request by cirno-999**
The Polish translation has been in quite a bad shape -- there was no source .txt file, only the .gxt blob (which was missing quite a bit of strings, too).

I took on the task of generating a `polish.txt` file based on `american.txt` (to later compile to .gxt with the gtx.exe utility in utils/gxt). I decompiled the gxt with gta3gxt, fix the mojibake, then some SQL and perl magic, and done.
I also translated the missing 20 or so strings. There were some technical terms, I tried to use native Polish counterparts where *reasonable*.

Strings that I translated:
```
[FEC_FRC] FREE CAM  -> SWOBODNA KAMERA
[FEC_IVP] INVERT PAD VERTICALLY  -> ODWRÓĆ OŚ PADA W PIONIE
[FEC_JDE] Detected joystick  -> Wykryto pada
[FEC_JOD] DETECT JOYSTICK  -> WYKRYJ PADA
[FEC_JPR] Press any key on the joystick of your choice that you want to use on the game, and it will be selected.  -> Naciśnij dowolny klawisz na padzie, którego chcesz użyć w grze.
[FED_AAS] ANTI ALIASING  -> WYGŁADZANIE KRAWĘDZI
[FED_BIL] BILINEAR  -> DWULINIOWE
[FED_CLF] COLOUR FILTER  -> FILTR KOLORU
[FED_FIL] TEXTURE FILTERING  -> FILTROWANIE TEKSTUR
[FED_FLS] FULLSCREEN  -> PEŁNY EKRAN
[FED_MBL] MOTION BLUR  -> ROZMYCIE RUCHU
[FED_MFX] MATFX  -> MATFX
[FED_MIP] MIP MAPPING  -> MIPMAPPING
[FED_NEO] NEO  -> PODŚWIETLENIE
[FED_PRM] PED RIM LIGHT  -> PODŚWIETLENIE PED
[FED_RGL] ROAD GLOSS  -> POŁYSK DROGI
[FED_TRL] TRILINEAR  -> TRÓJLINIOWE
[FED_VPL] VEHICLE PIPELINE  -> POTOK POJAZDÓW
[FED_WLM] WORLD LIGHTMAPS  -> LIGHTMAPY ŚWIATA
[FED_WND] WINDOWED  -> OKIENKOWY
[FEL_JAP] JAPANESE  -> JAPOŃSKI
[FEM_2PR] PS2 ALPHA TEST  -> ALFA TEST PS2
[FEM_CSB] CUTSCENE BORDERS  -> RAMKI CUTSCENEK
[FEM_HIG] HIGH  -> WYSOKIE
[FEM_ISL] MAP MEMORY USAGE  -> PRZYPISZ WYKORZYSTANIE PAMIĘCI
[FEM_LOW] LOW  -> NISKIE
[FEM_MED] MEDIUM  -> ŚREDNIE
[FEM_MOB] MOBILE  -> MOBILNY
[FEM_NRM] NORMAL  -> NORMALNY
[FEM_PS2] PS2  -> PS2
[FEM_SCF] SCREEN FORMAT  -> FORMAT OBRAZU
[FEM_SIM] SIMPLE  -> PROSTE
[FEM_XBX] XBOX  -> XBOX
[FESZ_RM] RETRY?  -> PONOWIĆ?
[FET_GRA] GRAPHICS SETUP  -> USTAWIENIA GRAFIKI
[FET_RMS] REPLAY MISSION  -> PONÓW MISJĘ
[YES] Yes  -> Tak
[Z] Z-axis value: ~1~  -> Wartość osi Z: ~1~
```
Also, I've updated build.bat.

### CODE REVIEW

File: utils/gxt/polish.txt [hunk 1]
```diff
@@ -0,0 +1,8072 @@
...
+~g~Nie rozdzielajcie się, niech cała grupa porusza się razem!
+	
+[HEY3]
+~g~Straciłeś z oczu swojego podopiecznego - wracaj i odszukaj 8-Balla!
+	
+[HEY4]
+~g~Jeżeli stracisz Misty, to Luigi zadba, abyś stracił życie! Wracaj po nią!
+	
+[HEY5]
+~g~One of the girls is AWOL, Go back and round her up!
```
Comment (ShFil119) [inline utils/gxt/polish.txt:29]: this one is missing

Comment (copyread) [reply utils/gxt/polish.txt:29]: That's how it is in the original game files. I'm not confident translating the story lines, but if you have a suggestion I will be happy to add it

Comment (cirno-999) [reply utils/gxt/polish.txt:29]: That's how it is in the original game files. I'm not confident enough to translate the story lines, but if you have a suggestion I'll be happy to add it

Comment (cirno-999) [reply utils/gxt/polish.txt:29]: hmm, is it not gonna get merged without this line? I would suggest `Jedna z dziewczynek została w tyle, wróć i ją zabierz!` if anything. Would go in the stylistic mood of the OG TL.

File: utils/gxt/polish.txt [hunk 2]
```diff
@@ -0,0 +1,8072 @@
...
+USTAWIENIA GRAFIKI
+	
+[FED_MIP]
+MIPMAPPING
+	
+[FED_AAS]
+WYGŁADZANIE KRAWĘDZI
+	
+[FED_FIL]
+
```
Comment (Sergeanur) [inline utils/gxt/polish.txt:7970]: Missing

Comment (cirno-999) [reply utils/gxt/polish.txt]: Added

**Comment by Sergeanur** [comment]
I've seen some critisim of it on the discord but no one bothered with a review, so I'll just merge it as is.

**Comment by cirno-999** [comment]
> I've seen some critisim of it on the discord but no one bothered with a review, so I'll just merge it as is.

Thanks! If anyone has any feedback I'll change stuff, no problem.

**Pull request merged**